### PR TITLE
feat(Analytics): Add initial processing such as Guild and User Count, as well as Voice connections and Command uses

### PR DESCRIPTION
**Pull request by Quantumlyy**

### CODE REVIEW

File: src/lib/SkyraClient.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { Client as VezaClient } from 'veza';
 import { Webhook } from 'discord.js';
 import { FSWatcher } from 'chokidar';
 import { DashboardClient } from 'klasa-dashboard-hooks';
+import { InfluxDB, WriteApi, WritePrecision } from '@influxdata/influxdb-client';
```
Comment (favna) [inline src/lib/SkyraClient.ts]:
```suggestion
import { InfluxDB, WriteApi, WritePrecision } from '@influxdata/influxdb-client';
import { Webhook } from 'discord.js';
import { FSWatcher } from 'chokidar';
import { DashboardClient } from 'klasa-dashboard-hooks';
```

This is basically what organize imports would make of it.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -25,15 +25,15 @@
...
 # QuantumlyTangled
-# Now while I may not have made all of them I would like to be notified of them so... this is what the following line is for
-/docker/ @quantumlytangled
 /src/commands/GameIntegration/ffxiv.ts @quantumlytangled
 /src/commands/Twitch/ @quantumlytangled
-/src/events/twitchStreamOffline.ts @quantumlytangled
-/src/events/twitchStreamOnline.ts @quantumlytangled
+/src/events/twitch/ @quantumlytangled
+/src/events/analytics/ @quantumlytangled
+/src/lib/types/Enums.d.ts @quantumlytangled
```
Comment (favna) [inline .github/CODEOWNERS]: Are you sure you want this? You're going to get GH pinged for things that do not particularly concern analytics.

**Review by favna — CHANGES_REQUESTED** [review]

File: src/events/klasaReady.ts [hunk 3]
```diff
@@ -39,7 +40,7 @@ export default class extends Event {
 	private async initPostStatsTask() {
 		const { tasks } = this.client.schedule;
 		if (!tasks.some(task => task.taskName === 'poststats')) {
-			await this.client.schedule.create('poststats', '*/15 * * * *', {});
+			await this.client.schedule.create('poststats', '*/10 * * * *', {});
```
Comment (favna) [inline src/events/klasaReady.ts]:
```suggestion
			await this.client.schedule.create('poststats', '*/15 * * * *', {});
```

Why change this to 10?

**Comment by Quantumlyy** [comment]
Here is the script to get total command usage.

```flux
from(bucket: "analytics")
  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
  |> filter(fn: (r) => r["_measurement"] == "commands")
  |> sum(column: "_value")
  |> group(mode:"by")
  |> sum(column: "_value")
```

**Pull request closed without merging**